extensions/sftp/processors/PutSFTP.cpp

```diff
@@ -260,9 +260,9 @@ bool PutSFTP::processOne(const std::shared_ptr<core::ProcessContext> &context, c
     remote_path = ".";
   }
   if (context->getDynamicProperty(DisableDirectoryListing.getName(), value)) {
```

Comment:
This pattern seems strange. Will ask the others on what this is meant to achieve.